File: TumeKyouen/Model/TumeKyouenModel.m
//
//  TumeKyouenModel.m
//  TumeKyouen
//
//  Created by 石倉 昇 on 2013/02/24.
//  Copyright (c) 2013年 noboru. All rights reserved.
//

#import "TumeKyouenModel.h"


@implementation TumeKyouenModel

@dynamic stageNo;
@dynamic clearDate;
@dynamic clearFlag;
@dynamic size;
@dynamic stage;
@dynamic creator;

- (NSString *)description {
    return [NSString stringWithFormat:@"stageNo = %@, size = %@, stage = %@", self.stageNo, self.size, self.stage];
}

@end

File: TumeKyouen/Database/TumeKyouenModel.swift
//
//  TumeKyouenModel.swift
//  TumeKyouen
//
//  Created by 石倉 昇 on 2016/02/01.
//  Copyright © 2016年 noboru. All rights reserved.
//

import CoreData

@objc(TumeKyouenModel)
class TumeKyouenModel: NSManagedObject {
    @NSManaged var stageNo: NSNumber
    @NSManaged var clearDate: Date
    @NSManaged var clearFlag: NSNumber
    @NSManaged var size: NSNumber
    @NSManaged var stage: String
    @NSManaged var creator: String
}
